errors/errors.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package errors provides an error type for Go Cloud APIs.
+package errors
+
+import "fmt"
+
+// A Code describes the error's category. Programs should act upon an error's
+// code, not its message.
+type Code int
+
+const (
+	// The error could not be categorized.
+	Unknown Code = iota
```

Comment:
Consider binding them to explicit constants to make sure they don't change?

Reply:
Done.